can/interfaces/kvaser/canlib.py

```diff
@@ -330,6 +345,16 @@ def init_kvaser_library():
             log.warning("Kvaser canlib could not be initialized.")
 
 
+def init_kvaser_library():
+    if __canlib is not None:
+        try:
+            log.debug("Initializing Kvaser CAN library")
+            canInitializeLibrary()
+            log.debug("CAN library initialized")
+        except:
+            log.warning("Kvaser canlib could not be initialized.")
+
+
```

Comment:
Isn't this a duplicate?

Reply:
My bad, I reverted commit ad11bf95 that introduced the (unwanted) change.